.eslintrc.json

```diff
@@ -57,7 +57,10 @@
     "jsdoc/require-returns-description": "error",
     "jsdoc/check-alignment": "error",
     "jsdoc/check-param-names": "error",
-    "react-hooks/exhaustive-deps": "error"
+    "react-hooks/exhaustive-deps": "error",
+    "sonarjs/todo-tag": "warn",
+    "sonarjs/redundant-type-aliases": "warn",
+    "sonarjs/no-nested-conditional": "warn"
```

Comment:
I changed this error to a warn since it would involve changing a lot of entries such as in [announcements.tsx](https://github.com/DataBiosphere/data-browser/blob/72f9cf377f41d1221f5936fd98a638eba9167885/app/components/common/Banner/components/Announcements/announcements.tsx#L20). @frano-m I'm not sure if we should change all of these to use chained if statements or if it's best just to disable this warn all together?

```
  return (
    <Fragment>
      {isSessionTimeout ? (
        <Banner onClose={clearSessionTimeout}>
          For your security, you have been logged out due to 15 minutes of
          inactivity.
        </Banner>
      ) : isSystemUnavailable ? (
        <Banner>
          One or more of the system components are currently unavailable.
          Functionality may be degraded.
        </Banner>
      ) : isSystemIndexing ? (
        <Banner>
          Data indexing in progress. Downloads and exports are disabled as
          search results may be incomplete.
        </Banner>
      ) : (
        generalAnnouncement
      )}
    </Fragment>
  );
```